lhotse/recipes/ami.py

```diff
@@ -588,6 +590,16 @@ def prepare_ami(
         else prepare_supervision_other(audio, annotations)
     )
 
+    # Some supervision segments exceed the recording bounds, so we remove them here
```

Comment:
Instead of doing this manually, you can run `recordings, supervisions = fix_manifests(recordings, supervisions)` at the end of processing

https://github.com/lhotse-speech/lhotse/blob/c42b181e728982f69ba1721959fe6f03078e821d/lhotse/qa.py#L40-L69

Reply:
Awesome!